aspnetcore/client-side/libman/libman-vs.md

```diff
@@ -0,0 +1,201 @@
+* Review the **Build** feed of the **Output** window for a LibMan activity log:
+
+  ```console
+  1>------ Build started: Project: LibManSample, Configuration: Debug Any CPU ------
+  1>
+  1>Restore operation started...
+  1>Restoring library jquery@3.3.1...
+  1>Restoring library bootstrap@4.1.3...
+  1>
+  1>2 libraries restored in 10.66 seconds
+  1>LibManSample -> C:\LibManSample\bin\Debug\netcoreapp2.1\LibManSample.dll
+  ========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
+  ```
+
+When the restore-on-build behavior is enabled, the *libman.json* context menu displays a **Disable Restore Client-Side Libraries on Build** option. Selecting this option removes the `Microsoft.Web.LibraryManager.Build` package reference from the project file. Consequently, the client-side libraries are no longer restored on each build.
+
+Regardless of the restore-on-build setting, you can manually restore at any time. To manually restore, select the **Restore Client-Side Libraries** option from the *libman.json* context menu.
+
+## Clean library files
+
+LibMan can remove all library files that were previously restored in Visual Studio. To clean, right-click the *libman.json* file in **Solution Explorer** and select **Clean Client-Side Libraries**. To prevent unintentional removal of non-library files, the clean operation doesn't delete whole directories. It only removes files that were included in the previous restore.
+
+Messages are sent to the status bar and the **Library Manager** feed of the **Output** window. For example:
+
+```console
+Clean libraries operation started...
+Clean libraries operation completed
+2 libraries were successfully deleted in 1.91 secs
+```
+
+## Uninstall library files
+
+To uninstall library files:
+
+* Open *libman.json*.
+* Position the caret inside the corresponding `libraries` object literal.
+* Click the light bulb icon that appears in the left margin, and select **Uninstall \<library_name>@\<library_version>**:
+
+  ![Uninstall library context menu option](_static/uninstall-menu-option.png)
+
+Alternatively, you can manually edit the *libman.json* file and save. The restore operation runs on save and removes the library files that are no longer part of the LibMan manifest.
+
+## Update library version
+
+To check for an updated library version:
+
+* Open *libman.json*.
+* Position the caret inside the corresponding `libraries` object literal.
+* Click the light bulb icon that appears in the left margin, and hover over **Check for updates**.
+
+LibMan checks for a library version newer than the version installed. The following things can occur:
+
+* A **No updates found** message is displayed if the latest version is already installed.
+* The latest stable version is displayed if not already installed.
+
+  ![Check for updates context menu option](_static/update-menu-option.png)
+
+* If a pre-release is available, which is newer than the installed version, the pre-release is displayed too.
```

Comment:
Seems like the parenthetical info is a critical condition. How about ...

> If a pre-release newer than the installed version is available, the pre-release is displayed, too.

(also, comma before "too" ... or perhaps drop it)

Reply:
For the sake of brevity, I'll drop "too". Thank you for the suggested text. I like yours much better.

Reply:
"pre-release" ... I'm thinking now that this is another one where they've have us follow Chicago and drop the hyphen.